hardware/setup/create_framework.tcl

```diff
@@ -283,6 +283,11 @@ if { ($fpga_card == "N250SP") && ($capi_bsp_dir != "not defined") } {
   add_files -fileset constrs_1 -norecurse $root_dir/setup/$fpga_card/snap_$fpga_card.xdc
 #  add_files -fileset constrs_1 -norecurse $root_dir/setup/$fpga_card/capi_bsp_pblock.xdc
 }
+if { ($fpga_card == "RCXVUP") && ($capi_bsp_dir != "not defined") } {
```

Comment:
Could be added to the statement above (the RCXVUP specifics are defined via environment variable `$fpga_card`)

Reply:
combined